src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -901,6 +901,181 @@ BOOST_AUTO_TEST_CASE(GradientFastLSTMLayerTest)
   BOOST_REQUIRE_LE(CheckGradient(function), 0.2);
 }
 
+/**
+ * Testing the overloaded Forward of the LSTM layer.
```

Comment:
Please make it `Forward()`. Also, it should be a comma rather than a full stop at the end of the sentence, and please try to make use of full 80 characters.

Reply:
Sure